### ISSUE TITLE
Gates/doors used to open slower

### ISSUE THREAD

**Issue by Nikolassparrow** [QuentinGruber/h1z1-server#2025]
**Describe the bug**
Not necessarily a bug, but more of a feature I remember and seeing in a lot of old videos. Right now, gates/doors open about 2x-3x quicker than their 2016 counterparts.

**To Reproduce**
1. Go to any door or place a deck foundation/ground tamper with a gate on it. 
2. Open and close the gate/door and observe the speed of the door/gate

**Expected behavior**
The door/gate should open slower and match their 2016 counterparts

**Screenshots**
https://www.youtube.com/watch?v=mUm95wfBj94 (1:10 ; 13:05)

**Comment by QuentinGruber** [comment]
yep i guess this is bc there was a specific packet for door opening but we didn't use it ( if you wanna dig that hole go for it :P )

**Comment by Nikolassparrow** [comment]
> yep i guess this is bc there was a specific packet for door opening but we didn't use it ( if you wanna dig that hole go for it :P )

I did do a little bit of research on it but I could never get anything close to working... I'm not sure how to reverse engineer the game too so finding the correct packet would have to be something for meme, jason or you to check out 😅

**Comment by QuentinGruber** [comment]
> > yep i guess this is bc there was a specific packet for door opening but we didn't use it ( if you wanna dig that hole go for it :P )
> 
> I did do a little bit of research on it but I could never get anything close to working... I'm not sure how to reverse engineer the game too so finding the correct packet would have to be something for meme, jason or you to check out 😅

the stuff that block me isn't the packet but like i guess we have to setup the npc as a door and i don't know how to do that :sweat:

**Comment by QuentinGruber** [comment]
> > > yep i guess this is bc there was a specific packet for door opening but we didn't use it ( if you wanna dig that hole go for it :P )
> > 
> > 
> > I did do a little bit of research on it but I could never get anything close to working... I'm not sure how to reverse engineer the game too so finding the correct packet would have to be something for meme, jason or you to check out 😅
> 
> the stuff that block me isn't the packet but like i guess we have to setup the npc as a door and i don't know how to do that 😓

probably a field in the npc schema but gl to find it

**Comment by QuentinGruber** [comment]
but for the speed we could just reduce the speed of the position update but won't fix what i told about door opening only one way

**Comment by QuentinGruber** [comment]
in the end noone care
